website/pages-parts/Tutorial/GuestSchema.ts

```diff
@@ -1,12 +1,20 @@
+// <schema>
 // <validator>
-import Ajv from 'ajv';
+import Ajv, { JSONSchemaType } from 'ajv';
 // </validator>
+// </schema>
```

Comment:
Is there an empty line between this `import` and the `interface` when rendered?

Reply:
Yes, there is.
<img width="1007" alt="image" src="https://user-images.githubusercontent.com/13087338/195990652-a40328b1-e22c-4008-9100-ae5fe70dcc9f.png">